packages/typescript/chat-core/test/ai/ai-provider.test.tsx

```
@@ -0,0 +1,171 @@
+          memoryLimit: 5,
+          enableActions: false,
+          enableSuggestions: true,
+        },
+      },
+    ];
+
+    configs.forEach((config) => {
+      expect(() => {
+        React.createElement(AIProvider, {
+          config,
+          children: React.createElement('div', null, 'Test'),
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use additional arguments to React.createElement

(lint/correctness/noChildrenProp)